packages/react/src/components/Portal/index.ts

```diff
@@ -1,22 +1,28 @@
-/**
- * Copyright IBM Corp. 2016, 2023
- *
- * This source code is licensed under the Apache-2.0 license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-import PropTypes from 'prop-types';
```

Comment:
Portal isn't within `/internal` but isn't exported, so I agree removing proptypes is a good option here 👍